docs/apache-airflow-providers-oracle/connections/oracle.rst

```diff
@@ -60,8 +63,35 @@ Extra (optional)
     * ``purity`` - one of ``new``, ``self``, ``default``. Specify the session acquired from the pool.
       configuration parameter.
 
-    More details on all Oracle connect parameters supported can be found in
-    `cx_Oracle documentation <https://cx-oracle.readthedocs.io/en/latest/module.html#cx_Oracle.connect>`_.
+    Connect using Dsn and Sid, Dsn and Service_name, or only Host `(OracleHook.getconn Documentation) <https://airflow.apache.org/docs/apache-airflow-providers-oracle/stable/_modules/airflow/providers/oracle/hooks/oracle.html#OracleHook.get_conn>`_.
+
+    For example:
+
+    .. code-block:: python
+
+        Host = "(DESCRIPTION=(ADDRESS=(PROTOCOL=TCP)(HOST=dbhost.example.com)(PORT=1521))(CONNECT_DATA=(SERVICE_NAME=orclpdb1)))"
+
+    or
+
+    .. code-block:: python
+
+        Dsn = "dbhost.example.com"
+        Service_name = "orclpdb1"
+
+    or
+
+    .. code-block:: python
+
+        Dsn = "dbhost.example.com"
+        Sid = "orcl"
+
+
+    More details on all Oracle connect parameters supported can be found in `cx_Oracle documentation
+    <https://cx-oracle.readthedocs.io/en/latest/api_manual/module.html#cx_Oracle.connect>`_.
+
+    Information on creating an Oracle Connection through the web user interface can be found in Airflow's `Managing Connections Documentation
+    <https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html>`_.
```

Comment:
Do we use http links for internal reference in our docs?
shouldn't this be  like the security section [here](https://github.com/apache/airflow/blob/master/docs/apache-airflow/howto/connection.rst#securing-connections) 
So I believe it should be:
:doc:`/howto/connection/index`

Reply:
You are correct. I will make the change suggested.

Reply:
@eladkal 
The changes caused this error: WARNING: unknown document: /howto/connection/index. Do you know why the doc link failed?

I reverted it to previous commit.

Reply:
The underlying error was because of this:
WARNING: unknown document: /howto/connection/index